addons/point_of_sale/static/src/app/components/popups/number_popup/number_popup.xml

```diff
@@ -10,10 +10,12 @@
                 </div>
             </div>
             <div t-if="props.feedback(this.state.buffer)" class="p-2 mx-auto mt-n2 mb-2" t-esc="props.feedback(this.state.buffer)" />
+            <t t-set="isValid" t-value="props.isValid(state.buffer)"/>
+            <div t-if="!isValid and props.isValidFeedback" class="p-2 mx-auto mt-n2 mb-2 text-center" t-esc="props.isValidFeedback(this.state.buffer)" />
             <Numpad buttons="props.buttons" class="'p-1 rounded-3 bg-200'"/>
             <t t-set-slot="footer">
                 <div class="d-flex flex-grow-1 justify-content-center">
-                    <button t-att-disabled="!props.isValid(state.buffer)" class="btn btn-primary btn-lg lh-lg o-default-button me-2" tabindex="1" t-on-click="confirm">
+                    <button t-att-disabled="!props.isValid(state.buffer) and props.isValidBlocking" class="btn btn-primary btn-lg lh-lg o-default-button me-2" tabindex="1" t-on-click="confirm">
```

Comment:
use the `isValid` that you set above

Reply:
I cannot use it in t-set the state isn't updated... :(